agent-chart/values.yaml

```diff
@@ -0,0 +1,40 @@
+# Use this only if you want to replace the default that is .Chart.Name as the name of all the objects.
+nameOverride: ""
+
+# If true, creates secret
+upstream:
+  createSecret: false
```

Comment:
@moshloop Should the default here be true ?

We do check all the necessary values in templates/required.yaml